ee/desktop/user/client/dialer_posix.go

```diff
@@ -9,7 +9,8 @@ import (
 )
 
 func dialContext(socketPath string) func(_ context.Context, _, _ string) (net.Conn, error) {
```

Comment:
Does the return need `ctx`? I'm not really sure how named vars work here

Reply:
I'm not sure, so I added just incase, it shouldn't hurt anything

Reply:
actually, nvm, it's not needed

https://go.dev/play/p/BPUDbYv-slf

Reply:
Thank you for checking